.github/workflows/tests-outerloop.yml

```diff
@@ -1,10 +1,30 @@
 # Executes outerloop tests
+#
+# COPILOT INSTRUCTIONS:
+# - Keep the 'paths:' list in sync across tests-outerloop.yml and
+#   tests-quarantine.yml
+# - Validate that each path exists in the repository before adding or
+#   updating the list
+# - No external YAML file is used—only the workflow YAMLs themselves
+#   hold the list
+#
 name: Outerloop Tests
 
 on:
   workflow_dispatch:
+
   schedule:
-    - cron: '0 2 * * *' # Daily at 02:00 UTC
+    - cron: '0 2 * * *'  # Daily at 02:00 UTC
+
+  # Trigger on pull requests that modify infrastructure or workflow files
+  pull_request:
+    paths:
+      - '.github/actions/**'
+      - '.github/workflows/**'
+      - 'eng/**'
+      - '!eng/pipelines/**'
+      - '!eng/scripts/**'
+      - '!eng/*pack/**'
```

Comment:
What about eng\common? Assuming we don't want to run this on arcade updates?

Reply:
Arcade updates can break this as we use arcade commands - `build.sh` - on these workflows.